straxen/plugins/veto_pulse_processing.py

```diff
@@ -13,7 +13,8 @@
         help='Save (left, right) samples besides hits; cut the rest'),
     strax.Option(
         'baseline_samples_nv',
-        default=10, track=True,
+        default_by_run=[(0, 10),
```

Comment:
Does it works with int? I'd had expected strings would be what you'd need.

Reply:
Yes, was also a bit surprised. But, you can check the lineage and/or here: https://github.com/XENONnT/straxen/blob/e996db70310559c37a87eec0796df012d3f4f7e0/straxen/plugins/peak_processing.py#L118-L119